recipes/crc_cpp/all/conanfile.py

```diff
@@ -0,0 +1,53 @@
+from conans import ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+required_conan_version = ">=1.33.0"
+
+
+class Crc_CppConan(ConanFile):
+    name = "crc_cpp"
+    description = "A header only, constexpr / compile time small-table based CRC library for C++17 or newer"
+    topics = ("conan", "crc_cpp", "crc", "constexpr" "cpp17", "cpp20", "header-only")
+    settings = "compiler", "os"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/AshleyRoll/crc_cpp"
+    license = "MIT"
+    no_copy_source = True
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _supported_compiler(self):
+        compiler = str(self.settings.compiler)
+        version = tools.Version(self.settings.compiler.version)
+        if compiler == "Visual Studio" and version >= "15":
+            return True
+        elif compiler == "gcc" and version >= "9":
+            return True
+        elif compiler == "clang" and version >= "5":
+            return True
+        elif compiler == "apple-clang" and version >= "10":
+            return True
+        else:
+            self.output.warn("{} recipe lacks information about the {} compiler standard version support".format(self.name, compiler))
+        return False
+
+    def configure(self):
```

Comment:
```suggestion
    def validate(self):
```

Those checks are better in `validate` method